### PR TITLE
Ft/add admin auth

### PR DESCRIPTION
**Pull request by rhiroyuki**

### CODE REVIEW

File: app/models/admin.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
```
Comment (CaiqueMitsuoka) [inline app/models/admin.rb]: Remove comments

Comment (rhiroyuki) [reply app/models/admin.rb]: Done. See https://github.com/FatecSorocaba/callforpapersXXIV/pull/13/commits/ec21ef60c899f86d621dcebb24f49dc195fcc630.
Could you review it again entirely? I've done some refactoring.

**Comment by rhiroyuki** [comment]
I've done some refactoring... devise now handles admin creation process.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -13,6 +13,14 @@
     <header class="header">
       <h1>Call 4 Papers</h1>
     </header>
+    <% if admin_signed_in? %>
+      <section>
+        Logado como: <%= current_admin.email %>
+        <%= link_to 'Sign out', destroy_admin_session_path, method: :delete %> 
+        <%= link_to 'New Admin', new_admin_registration_path %>
+        <%= link_to 'Tasks', speakers_path %>
```
Comment (guivinicius) [inline app/views/layouts/application.html.erb]: Tasks or Speakers ?

File: config/locales/devise.en.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
```
Comment (guivinicius) [inline config/locales/devise.en.yml:1]: What about the pt-br file ?

File: config/initializers/devise.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Devise.setup do |config|
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (guivinicius) [inline config/initializers/devise.rb]: I think you should change this for another email

File: config/locales/pt-BR.yml [hunk 5]
```diff
@@ -0,0 +1,265 @@
+pt-BR:
```
Comment (guivinicius) [inline config/locales/pt-BR.yml]: This is fine for now, but you could split into smaller files just like we have into app folder.

models/
views/

for specific files

and for the rails default translations you could add them into a `default.pt-br.yml`

**Pull request merged**